### ISSUE TITLE
Stopping following caused by twitter lazy load

### ISSUE THREAD

**Issue by raphacmartin** [tlemens/twitter-mass-follow#39]
Sometimes, the message "no more profiles found" appear, so I just scroll the page down to load more profiles and click in follow again.
To prevent it, the plugin could scroll down by it self, I am used to paste this code in console to solve this:

```
setInterval(function(){
   window.scrollTo(0, $('#page-container').height());
}, 2000)
```

Thanks for the extension!

**Comment by tlemens** [comment]
Would you mind adding it and create a PR? So in case no more profiles found, it will try to scroll down once. It actually should automatically by focusing the last button. Did you find out when it actually happens? Can you reproduce it?

NOTE: we have to use plain js since we don't want a jquery dependency in this project to keep the size small

**Comment by soulpull** [comment]
yeah,it can't working with issue message "no more profiles found" appear
